app_dart/lib/src/request_handlers/get_branches.dart

```diff
@@ -55,7 +56,9 @@ class GetBranches extends RequestHandler<Body> {
     List<Branch> branches = await datastore
         .queryBranches()
         .where(
-          (Branch b) => DateTime.now().millisecondsSinceEpoch - b.lastActivity! < kActiveBranchActivity.inMilliseconds,
+          (Branch b) =>
+              DateTime.now().millisecondsSinceEpoch - b.lastActivity! < kActiveBranchActivity.inMilliseconds ||
+              <String>['main', 'master'].contains(b.name),
```

Comment:
Nit: define a default branch list in config?

Reply:
We already have multiple properties with config names in branches, adding one more will make it confusing. Also this list will never grow and eventually will be only main.